modules/templates/util_render.go

```diff
@@ -224,3 +224,25 @@ func RenderLabels(ctx context.Context, labels []*issues_model.Label, repoLink st
 	htmlCode += "</span>"
 	return template.HTML(htmlCode)
 }
+
+func RenderDependencies(ctx context.Context, dependencies []*issues_model.DependencyInfo) template.HTML {
```

Comment:
Does it really need to introduce a function?

I can see this function's purpose is just "rendering HTML code", so could a sub-template do the work and work better?

Reply:
No its not possible, because of RTL support
The template.HTML-string is given to the translator for `depends on: %s`

Reply:
Could you help to provide some examples for how RTL would be used in this case (eg: some example translations) ?

I didn't know it well, and IIRC Gitea didn't work well for RTL in some cases before.

I think it's good to pay more attention to RTL cases, so it's better to document it somewhere. Otherwise, such code might be refactored into "tmpl" files if other people forget that the code was written for RTL.

Reply:
I don't know any translations because I only speak LTR languages

But I changed it because of https://github.com/go-gitea/gitea/pull/29117#discussion_r1484588107

Reply:
@silverwind what do you think about this? Using backend generated HTML content is quite difficult to maintain. If there is a chance, could it use the tmpl files to render?

Reply:
I suggested this actually because the template code would have gotten too messy otherwise with nested subtemplates etc. Not saying it couldn't be done and it should ideally be attempted in a separate file maybe.